doc/aspice_swe3_4/example/iceoryx_component/CMakeLists.txt

```diff
@@ -42,25 +42,25 @@ iox_add_library(
 ########## test ##########
 #
 
-if(BUILD_TEST AND NOT GTest_FOUND)
```

Comment:
Is this component `aspice_swe3_4` still needed or would it be possible to remove it?

Reply:
@elfenpiff It's need to the doxygen generation and meant as an example on how to achieve traceability. However, it's quite outdated and we need to update it.